hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/downloader/ConsensusValidatorImplTest.java

```diff
@@ -368,14 +388,13 @@ void testSignaturesVerifiedWithFullConsensusRequired() {
     @SneakyThrows
     @Test
     void testFailedVerificationSignatureConsensus() {
-        var nodeKeyPair = KeyPairGenerator.getInstance("RSA").generateKeyPair();
-        var publicKey = nodeKeyPair.getPublic();
-        var nodeAccountIDPubKeyMap = new HashMap<String, PublicKey>();
-        nodeAccountIDPubKeyMap.put("0.0.3", publicKey);
-        nodeAccountIDPubKeyMap.put("0.0.4", publicKey);
-        nodeAccountIDPubKeyMap.put("0.0.5", publicKey);
-        nodeAccountIDPubKeyMap.put("0.0.6", publicKey);
-        when(currentAddressBook.getNodeAccountIDPubKeyMap()).thenReturn(nodeAccountIDPubKeyMap);
+        // Zero node stakes occurs when falling back to counting signatures
+        nodeStakes(0, 0, 0, 0);
+
+        var entity6 = new EntityId(0L, 0L, 6L, EntityType.ACCOUNT);
```

Comment:
nit, can simplify it here and everywhere 0 shard and realm are used:
```suggestion
        var entity6 = new EntityId(6L, EntityType.ACCOUNT);
```

Reply:
Great thanks, I have made these changes.